README.md

```diff
@@ -59,18 +59,16 @@ The prebuilt image and document are here:
 # How to Build:
 
 ## Windows
-- `git clone https://github.com/nxp-imx/mfgtools.git`
+- `git clone --recurse-submodules https://github.com/nxp-imx/mfgtools.git`
 - `cd mfgtools`
-- `git submodule init`
-- `git submodule update`
 - `open msvs/uuu.sln with Visual Studio 2017`
 
 Visual Studio
 
 Note that, since uuu is an OSI compliant Open Source project, you are entitled to download and use the freely available Visual Studio Community Edition to build, run or develop for uuu. As per the Visual Studio Community Edition license this applies regardless of whether you are an individual or a corporate user.
 
 ## Linux
-- `git clone https://github.com/nxp-imx/mfgtools.git`
+- `git clone --recurse-submodules https://github.com/nxp-imx/mfgtools.git`
```

Comment:
Linux and macOS needn't --recurse-submodules. uuu will use system library to build.

Reply:
I removed --recurse-submodules on Linux and macOS